index.js

```diff
@@ -44,7 +44,9 @@ class Conf {
 		this.encryptionKey = options.encryptionKey;
 
 		const fileExtension = options.fileExtension ? `.${options.fileExtension}` : '';
-		this.path = path.resolve(options.cwd, `${options.configName}${fileExtension}`);
+		this.path = path.resolve(options.cwd, `${options.configName}.${options.fileExtension}`);
+    
+		this.validate = options.validate;
```

Comment:
```suggestion
		this._validate = options.validate;
```

It's internal and should be marked as such.